fxa-oauth-server/lib/config.js

```diff
@@ -52,6 +52,12 @@ const conf = convict({
     env: 'OAUTH_CLIENTS',
     default: []
   },
+  clientManagementEnabled: {
```

Comment:
I like the idea of having top level config be `clientManagement` and have a nested `enabled` config. Similar to how it is done in [auth-server](https://github.com/mozilla/fxa-auth-server/blob/d98b10ca30b44e7f2faabe101a5b999ab527a1c6/config/index.js#L747). It could also be useful if there are eventually any other options that go in `clientManagement`.